protocol/x/clob/flags/flags.go

```diff
@@ -33,8 +34,9 @@ const (
 
 // Default values.
 const (
-	DefaultMaxLiquidationOrdersPerBlock    = math.MaxUint32
-	DefaultMaxDeleveragingAttemptsPerBlock = 35
+	DefaultMaxLiquidationOrdersPerBlock        = 20
+	DefaultMaxDeleveragingAttemptsPerBlock     = 5
+	DefaultMaxDeleveragingSubaccountsToIterate = 500
```

Comment:
For my understanding, what methodology should people use for configuring these values in the future?

Fine with creating a medium priority M6 ticket for research to figure this out later.